### ISSUE TITLE
Advanced Table List template - Enhancements

### ISSUE THREAD

**Issue by kiranetph** [Serhioromano/cobalt-9#154]
The commercial "Advanced Table" template has already some very helpful features like putting several fields into 1 column. Unfortunately it allows this feature only for 1 column.

It would be cool to define the fields per column.

How could this be done?
Maybe by defining it in a text area field.. 
Example:
![image](https://user-images.githubusercontent.com/15226809/65856136-5955be80-e393-11e9-963e-0c9173f71778.png)

I'm sure there are better ways to do it. Maybe subject for discussion.

**Comment by Serhioromano** [comment]
Good idea. And in time.

**Comment by kiranetph** [comment]
low priority
it can even be enhanced with Header / content alignment...
also the case, when 2 fields will be displayed side by side , etc 
Still a lot of room for discussion

Example for 3 fields( date and 2x upload) in a column on 2 rows 
![image](https://user-images.githubusercontent.com/15226809/65869095-43082c80-e3ac-11e9-8e7d-4ebf27aee1fe.png)

**Comment by KEZEO** [comment]
I agree with you.  Even in full view, to be able to align field according left, center or right
It was the problem for beginner.
Thank you
